{{cookiecutter.repo_name}}/CHANGELOG.rst

```diff
@@ -1,8 +1,8 @@
 Change Log
-----------
+==========
```

Comment:
What was the TOC problem that prompted this?  The section markup was shifted down one level from the other files so that it gets formatted properly when appended to the README for the PyPI description: [code](https://github.com/edx/cookiecutter-django-app/blob/master/%7B%7Bcookiecutter.repo_name%7D%7D/setup.py#L43), [example](https://pypi.python.org/pypi/django-user-tasks/0.1.4).  An example of RTD docs generated from this template as it has been: http://django-user-tasks.readthedocs.io/en/latest/